src/commands/executeFunction/EventGridCodeLensProvider.ts

```diff
@@ -0,0 +1,18 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { CodeLens, Range, type CodeLensProvider } from 'vscode';
+import { localize } from '../../localize';
+
+export class EventGridCodeLensProvider implements CodeLensProvider {
+    public provideCodeLenses(): CodeLens[] {
+        const sendRequestLens = new CodeLens(new Range(0, 0, 0, 0));
```

Comment:
As mentioned offline, it might be helpful to have this code lens also displayed at the bottom of the file. I think it would make it more obvious to the user, as well be convenient if you've scrolled to the bottom of the file and don't know that you need to scroll back to the top.

Reply:
Discussed offline. Ended up adding a line at the end then removing it 😛